### PR TITLE
Update D8 Components and Core, remove altered files based

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: lib/Drupal/Core/KeyValueStore/DatabaseStorage.php [hunk 1]
```diff
@@ -1,169 +0,0 @@
...
-
-  /**
-   * Implements Drupal\Core\KeyValueStore\KeyValueStoreInterface::deleteAll().
-   */
-  public function deleteAll() {
-    $this->connection->delete($this->table)
-      ->condition('collection', $this->collection)
-      ->execute();
-  }
-}
```
Comment (fabianx-ai) [inline lib/Drupal/Core/KeyValueStore/DatabaseStorage.php:169]: Is this file changed?

Comment (fabianx-ai) [inline lib/Drupal/Core/KeyValueStore/DatabaseStorage.php:169]: nvm, me - D8 core does not have a subtree-split yet ...

Comment (fabianx-ai) [inline lib/Drupal/Core/KeyValueStore/DatabaseStorage.php:169]: Anyway, is this file changed?

File: lib/Drupal/Core/KeyValueStore/DatabaseStorageExpirable.php [hunk 2]
```diff
@@ -1,166 +0,0 @@
-<?php
```
Comment (fabianx-ai) [inline lib/Drupal/Core/KeyValueStore/DatabaseStorageExpirable.php:1]: Is this file changed?

File: lib/Drupal/Core/StringTranslation/TranslationWrapper.php [hunk 3]
```diff
@@ -1,138 +0,0 @@
-<?php
```
Comment (fabianx-ai) [inline lib/Drupal/Core/StringTranslation/TranslationWrapper.php:1]: Lets leave this in here unchanged for now.

**Comment by drupol** [comment]
Updated.

**Pull request merged**